### DISCUSSION TITLE
leafmap add_velocity function on streamlit app

### DISCUSSION THREAD

**Discussion by xiaogangzhu** [opengeos/leafmap#1024] [Q&A] [answered]
Hi,

I am wondering if the add_velocity function in leafmap is able to draw on streamlit app? something like this

import leafmap.leafmap as leafmap

filename = "wind_global.nc"
m = leafmap.Map(layers_control=True)
m.add_basemap("CartoDB.DarkMatter")
m.add_velocity(
    filename,
    zonal_speed="u_wind",
    meridional_speed="v_wind",
    color_scale=[
        "rgb(0,0,150)",
        "rgb(0,150,0)",
        "rgb(255,255,0)",
        "rgb(255,165,0)",
        "rgb(150,0,0)",
    ],
)
m.to_streamlit(height=700)

**Comment by giswqs** [comment, marked as answer]
Unfortunately, no. Streamlit doesn't support ipyleaflet
